README.md

```diff
@@ -24,6 +24,25 @@ One could also use a standard Clang installation, build a sysroot from the
 sources mentioned above, and compile with
 "--target=wasm32-wasi --sysroot=/path/to/sysroot".
 
+## Install
+
+A typical installation from the release binaries might look like the following:
+```shell script
+wget https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-[version]/wasi-sdk-[version]-linux.tar.gz
+tar xvf wasi-sdk-[version]-linux.tar.gz
+cp -r wasi-sdk-[version] /opt/wasi-sdk
```

Comment:
Sorry I meant the SDK doesn't have to live in this location.   `/opt/wasi-sdk` is where it lives in the debian package but users of the tar ball don't need to put it there.   Indeed, I think I would recommend just using it within your $HOME, or wherever you untar it.

Reply:
Ok, I left the `cp` in to make it more obvious where `[WASI_SDK_PATH]` is coming from.

Reply:
...and I just resolved the misspellings 😄